gtsfm/configs/deep_front_end.yaml

```diff
@@ -10,7 +10,6 @@ SceneOptimizer:
     detector_descriptor:
       _target_: gtsfm.frontend.detector_descriptor.superpoint.SuperPointDetectorDescriptor
 
-
```

Comment:
unrelated change

Reply:
Causes a mypy error... but I can add it back